jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -383,6 +383,38 @@ def test_population_count(self, harness: primitive_harness.Harness):
     self.ConvertAndCompare(harness.dyn_fun, *harness.dyn_args_maker(self.rng()),
                            expect_tf_exceptions=True)
 
+  @primitive_harness.parameterized(primitive_harness.lax_add_mul)
+  def test_add_mul(self, harness: primitive_harness.Harness):
+    expect_tf_exceptions = False
+    dtype = harness.params["dtype"]
+    f_name = harness.params["f_jax"].__name__
+
+    if not dtype in [dtypes.bfloat16, np.float16, np.float32, np.float64, np.uint8,
+                     np.int8, np.uint16, np.int16, np.int32, np.int64, np.complex64,
+                     np.complex128]:
+      # TODO(bchetioui): tf.math.multiply is only defined for the above types.
+      expect_tf_exceptions = True
+    elif dtype is np.uint16 and f_name == "add":
+      # TODO(bchetioui): tf.math.add is defined for the same types as multiply,
+      # except uint16.
+      expect_tf_exceptions = True
+    self.ConvertAndCompare(harness.dyn_fun, *harness.dyn_args_maker(self.rng()),
+                           expect_tf_exceptions=expect_tf_exceptions)
+
+  @primitive_harness.parameterized(primitive_harness.lax_min_max)
+  def test_min_max(self, harness: primitive_harness.Harness):
+    expect_tf_exceptions = False
+    dtype = harness.params["dtype"]
+
+    if not dtype in [dtypes.bfloat16, np.float16, np.float32, np.float64, np.uint8,
+                     np.int16, np.int32, np.int64]:
+      # TODO(bchetioui): tf.math.maximum and tf.math.minimum are only defined for
```

Comment:
Here are we missing: int8, uint16, uint32 and uint64?

Reply:
Addressed in `bfc8b789`